.github/workflows/ci.yml

```
@@ -35,7 +38,14 @@ jobs:
-          sudo apt install -y jq yq
+          sudo apt install -y jq
+
+          curl -sSL https://github.com/mikefarah/yq/releases/download/v4.45.1/yq_linux_amd64 -o /usr/local/bin/yq
+          chmod +x /usr/local/bin/yq
+
+          cargo install cargo-set-version
       - name: Run Tests
         run: |
           task test
+      - name: Increase Version
+        uses: ./
```

actionlint (1.7.4)

missing input "github_token" which is required by action "version-update" defined at "./". all required inputs are "github_token"

(action)